model/Core/Properties/countyCode.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# countyCode
+
+## Summary
+
+A code that identifies a county. A county is a territorial division in some countries, forming the chief unit of local administration. In the US, a county is a political and administrative division of a state.
+
+## Description
+
+A code that identifies a county. A county is a territorial division in some countries, forming the chief unit of local administration. In the US, a county is a political and administrative division of a state.
```

Comment:
```suggestion
A code that identifies a county. A county is a territorial division in some countries, often forming a primary unit of local administration.

In the US, a county is a political and administrative division of a state.
```

Add "often" to soften the claims, as this could be vary by countries.

For the US, do we like to specify the Federal Information Processing Standard Publication 6-4 (FIPS 6-4) for county code?